src/main/java/com/example/EnumDemo/controller/DressController.java

```diff
@@ -0,0 +1,47 @@
+package com.example.EnumDemo.controller;
+
+import com.example.EnumDemo.dto.DressAdditionDto;
+import com.example.EnumDemo.dto.DressDisplayDto;
+import com.example.EnumDemo.enumPackage.Color;
+import com.example.EnumDemo.enumPackage.DressType;
+import com.example.EnumDemo.enumPackage.Size;
+import com.example.EnumDemo.repository.DressesRepository;
+import com.example.EnumDemo.services.DressService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/dresses")
+public class DressController {
+    @Autowired
+    DressService dressService;
+    @GetMapping
+    public List<DressDisplayDto> getAllDresses(@RequestParam(name = "Brand", required = false)String[] brands,
+                                               @RequestParam(name = "Size", required = false)Size size,
+                                               @RequestParam(name = "Type", required = false)DressType type,
+                                               @RequestParam(name = "Color", required = false)Color color,
+                                               @RequestParam(name = "Exclude color", required = false)Color Nocolor,
+                                               @RequestParam(name = "Price", required = false)Long price,
+                                               @RequestParam(name = "Lower Price", required = false)Long price1,
+                                               @RequestParam(name = "Higher Price", required = false)Long price2){
```

Comment:
```suggestion
                                               @RequestParam(name = "higherPrice", required = false)Long price2){
```
The name of the request params should be in camelcase.

Reply:
Done